config/l5-swagger.php

```diff
@@ -95,6 +95,36 @@
             'excludes' => [],
         ],
 
+        'scanOptions' => [
+            /**
+             * Custom query path processors classes.
+             *
+             * @link https://github.com/zircote/swagger-php/tree/master/Examples/schema-query-parameter-processor
+             * @see \OpenApi\scan
+             */
+            'processors' => [
+                // \App\SwaggerProcessors\SchemaQueryParameter::class,
+            ],
+
+            /**
+             * pattern: string       $pattern File pattern(s) to scan (default: *.php) .
+             * @see \OpenApi\scan
+             */
+            'pattern' => null,
+
+            /**
+             * analyser: defaults to \OpenApi\StaticAnalyser .
+             * @see \OpenApi\scan
+             */
+            'analyser' => null,
+
+            /**
+             * analysis: defaults to a new \OpenApi\Analysis .
+             * @see \OpenApi\scan
+             */
+            'analysis' => null,
```

Comment:
Why these 2 config keys are never used?

Reply:
These keys are allowed by `OpenApi\scan` method. So they are added in case someone wants to pass some data to the method. More freedom.

What concerns tests, I need to learn how to do this. I have low experience with writing tests.

Reply:
The problem is that you are not passing these to the `OpenApi\scan` You only pass `pattern` and `processors `

Reply:
Ooups... I've forgotten to pass them as well. Sorry :-)